IronSoftware.Drawing/IronSoftware.Drawing.Common/Color.cs

```diff
@@ -4,33 +4,42 @@
 
 namespace IronSoftware.Drawing
 {
+    // <summary>
+    /// <para>A universally compatible Color for .NET 7 and .NET 6, .NET 5, .NET Core. Windows, NanoServer, IIS,  macOS, Mobile, Xamarin, iOS, Android, Google Compute, Azure, AWS and Linux compatibility.</para>
```

Comment:
Change:
`    /// <para>A universally compatible Color for .NET 7 and .NET 6, .NET 5, .NET Core. Windows, NanoServer, IIS,  macOS, Mobile, Xamarin, iOS, Android, Google Compute, Azure, AWS and Linux compatibility.</para>`
To:
`    /// <para>A universally compatible Color for .NET 7, .NET 6, .NET 5, and .NET Core. As well as compatiblity with Windows, NanoServer, IIS,  macOS, Mobile, Xamarin, iOS, Android, Google Compute, Azure, AWS, and Linux.</para>`